dts/msm8916/msm8916-coolpad-coolpady80d.dts

```diff
@@ -0,0 +1,24 @@
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	#address-cells = <0x02>;
+	#size-cells = <0x02>;
+	qcom,msm-id = <0xce 0x00>;
+	qcom,board-id = <0x08 0x00>;
```

Comment:
```suggestion
	qcom,msm-id = <206 0>;
	qcom,board-id = <8 0>;
```

By the way it seems to be msm8916-mtp and would be better to merge this dts into `msm8916-mtp.dts`